### PR TITLE
Fixed Keyboard Buttons

### PR DESCRIPTION
**Pull request by AshciR**
The keyboard button was buttons weren't working because
the InlineKeyboardMarkup() method expected a List[List]
argument instead of a List

### CODE REVIEW

File: gerousiabot/bot_handlers.py [hunk 1]
```diff
@@ -65,25 +94,9 @@ def format_user_list(user_list: list) -> str:
...
 
 
-def start(update, context) -> None:
-    """Handler definition for the bot start command: /start"""
-    context.bot.send_message(chat_id=update.effective_chat.id, text="Gerousia bot! 🤖")
-    kb_markup = InlineKeyboardMarkup(start_keyboard())
-    update.message.reply_text('Please select one:', reply_markup=kb_markup)
-
-
-def check_users(update, context) -> None:
```
Comment (AshciR) [inline gerousiabot/bot_handlers.py:75]: Moved these up in the file for readability

File: gerousiabot/bot_handlers.py [hunk 2]
```diff
@@ -13,37 +18,52 @@ def get_bot_handlers() -> list:
...
+    kb_markup = InlineKeyboardMarkup(get_keyboard())
+    update.message.reply_text('Please select one:', reply_markup=kb_markup)
 
 
-def start_keyboard() -> list:
+def get_keyboard() -> list:
     """Initialize the keyboard"""
-    keyboard = [InlineKeyboardButton("Check Users", callback_data=Response.CHECK_USERS),
-                InlineKeyboardButton("Blank button", callback_data=Response.BLANK)]
+    keyboard = [[
```
Comment (AshciR) [inline gerousiabot/bot_handlers.py:39]: [Focus Point] This is what caused the bug. `InlineKeyboardMarkup()` expected a 2D array

File: gerousiabot/bot_handlers.py [hunk 3]
```diff
@@ -55,6 +75,15 @@ def get_user_list() -> list:
     return user_list
```
Comment (rick1e) [inline gerousiabot/bot_handlers.py:77]: Sees check_user function .... does not see check_user function being used.

Sees duplication of code in check_user function below if query.data == Response.CHECK_USERS.name .

Maybe i need sleep but this looks off.

File: tests/test_bot_handlers.py [hunk 4]
```diff
@@ -45,27 +50,31 @@ def test_unknown():
...
-@patch("gerousiabot.bot_handlers.Response", Test_Response)
-def test_keyboard_button_pressed(_input, expected):
+    keyboard = bot_handlers.get_keyboard()
+    assert len(keyboard) == 1
+    assert len(keyboard[0]) == 2
+    assert isinstance(keyboard, list)
+
+
+@pytest.mark.parametrize(
+    "_input", [test_response.CHECK_USERS.name, test_response.BLANK.name],
```
Comment (rick1e) [inline tests/test_bot_handlers.py:64]: No tests for the else

**Review by rick1e — COMMENTED** [review]
Can work but would suggest changes be made.

File: gerousiabot/bot_handlers.py (hunk 3 above)
Comment (jheuv) [reply gerousiabot/bot_handlers.py:77]: Original intent was to call that function in response to the keyboard button or a direct /checkusers command. After I implemented the button logic I had forgotten to remove the redundancy 🤦🏾‍♂️

File: tests/test_bot_handlers.py (hunk 4 above)
Comment (AshciR) [reply tests/test_bot_handlers.py:64]: Addressed

File: tests/test_telegram_bot.py [hunk 5]
```diff
@@ -25,32 +26,13 @@ def test_run_bot(mocked_get_api_token, mocked_get_bot_handlers, mocked_configure
...
 @patch("gerousiabot.telegram_bot.Updater")
 def test_configure_bot(mocked_updater):
     mocked_config = Mock()
     handler = Mock()
     mocked_updater.return_value = mocked_config
 
-    telegram_bot.configure_bot('test', [handler])
+    telegram_bot.configure_bot_handlers('test', [handler])
 
     mocked_config.dispatcher.add_handler.assert_called()
```
[the remark this answers is not in the document]
Comment (AshciR) [reply tests/test_telegram_bot.py:38]: Addressed

**Review by rick1e — APPROVED** [review]

**Pull request merged**